tests/suites/agents/task.sh

```diff
@@ -0,0 +1,20 @@
+test_agents() {
+    if [ "$(skip 'test_agents')" ]; then
+        echo "==> TEST SKIPPED: agent tests"
+        return
+    fi
+
+    set_verbosity
+
+    echo "==> Checking for dependencies"
+    check_dependencies juju
+
+    file="${TEST_DIR}/test-agents.txt"
+
+    JUJU_AGENT_TESTING_OPTIONS=CHARM_REVISION_UPDATE_INTERVAL=15s \
```

Comment:
How do we deal with the situation of running tests against an existing controller? It isn't going to work here.

Reply:
No it's not. It was designed to work when bootstrapping a new controller. There's not really a nice way to weave this into an existing controller since it needs an agent restart etc. We could do a big, bespoke ssh script. But the CI tests on Jenkins all use a new controller IIANM.